### PR TITLE
Add an initial README.md.

### PR DESCRIPTION
**Pull request by pbos**
Mentions basic usage of gtest-parallel as well as a section on flakiness
testing. This should be a lot more friendly to new users or anyone
stumbling upon this page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# gtest-parallel
+
+_This is not an official Google product._
+
+`gtest-parallel` is a script that executes [Google
+Test](https://github.com/google/googletest) binaries in parallel, providing
+good speedup on multi-core machines, especially for single-threaded tests, or
+tests that do otherwise not run at 100% CPU.
+
```
Comment (kwiberg) [inline README.md]: Maybe this is easier to parse?

"... providing good speedup for single-threaded tests (on multi-core machines) and tests that do not run at 100% CPU (on single- or multi-core machines)."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+The script works by listing all tests per-binary, and then executing them on
```
Comment (kwiberg) [inline README.md]: "listing the tests of each binary, and then"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    $ ./gtest-parallel path/to/binary... --gtest_filter=Foo.*:Bar.*
+
+This filter takes the same parameters as Google Test, so -Foo.\* can be used for
+test exclusion as well. This is especially useful for slow tests (that you're
+not working on), or tests that may not be able to run in parallel.
+
+## Flakiness
+
+Flaky tests (tests that may pass or fail, often even if the underlying code is
+correct) often cause a lot of developer pain. A test that fails only 1% of the
```
Comment (kwiberg) [inline README.md]: "(tests that do not deterministically pass or fail)"

**Review by kwiberg — APPROVED** [review]
Good initiative!

Some comments---take them or leave them.

File: README.md (hunk 1 above)
Comment (pbos) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (pbos) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (pbos) [reply README.md]: Done, thanks!

**Pull request merged**